contracts/ERC20SimpleSwap.sol

```diff
@@ -250,17 +255,10 @@ contract SimpleSwap {
     require(msg.sender == issuer, "SimpleSwap: not issuer");
     /* ensure we don't take anything from the hard deposit */
     require(amount <= liquidBalance(), "SimpleSwap: liquidBalance not sufficient");
-    issuer.transfer(amount);
+    token.transfer(issuer, amount);
     emit Withdraw(amount);
```

Comment:
This event is no longer necessary. We can just watch the token `Transfer` event.

Reply:
Removed